src/utils/experiments/checkUserDevice.ts

```diff
@@ -0,0 +1,21 @@
+export enum UserDeviceType {
+	Desktop = 'desktop',
+	Mobile = 'mobile',
+	Tablet = 'tablet',
+	Unknown = 'unknown',
+}
+
+export default function checkUserDevice(): UserDeviceType {
+	let userDevice = UserDeviceType.Unknown;
+
+	// calculate user device type based on maximum width values per device
+	if (window.matchMedia('(max-width: 767px)').matches) {
+		userDevice = UserDeviceType.Mobile;
+	} else if (window.matchMedia('(min-width: 768px) and (max-width: 991px)').matches) {
```

Comment:
Hmm, makes me wonder how this will behave on UCP with the video player. I think at 768px+ its considered desktop. I'm wondering if at these resolutions it WILL load the player.